portfolio-jsm/src/components/ProjectCard.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import React from "react";
+import { useMediaQuery } from "react-responsive";
+import Button from "./Button";
+import { ArrowRight } from "iconsax-react";
+import { ProjectCardProps } from "../types";
+
+const ProjectCard = ({ bgcolor, title }: ProjectCardProps) => {
+  const isMobile = useMediaQuery({ maxWidth: 1024 });
+
+  return (
+    <main
+      className={`${bgcolor} h-[514px] w-[90%] rounded-[10px] text-white-900 md:h-[440px] md:w-[1270px]`}
+    >
+      <div className="flex flex-col md:flex-row">
+        <div className="ml-6 mt-12 flex flex-col md:ml-24 md:mt-20 md:w-[50%]">
+          <h1 className="text-32 font-extrabold md:text-48">{title}</h1>
+          <div className="mt-3 flex flex-row">
+            <Button
+              buttonColor="bg-[#FFFFFF33] rounded-[6px] py-3"
+              textColor="text-white-900 font-semibold"
+              hPadding="px-3"
+              width={""}
+              height={""}
```

Comment:
Try making the props optional in the Button interface so you don't need to include things you're not using. You're already doing so with some of the props.